src/ir/module.h

```diff
@@ -83,7 +83,7 @@ class Block {
 
   Block() : module_(nullptr) {}
 
-  // Disable copy (and move) semantics.
+  // Disable copy semantics.
```

Comment:
When you remove copy semantics, it also disables move.

Reply:
This was a misfire in updating the comment, meant to update the other one. I've backed this change out.